cypress/e2e/marc/marc-authority/marc-authority-browser-search-resetAll-button.cy.js

```diff
@@ -0,0 +1,46 @@
+import Permissions from '../../../support/dictionary/permissions';
+import MarcAuthorities from '../../../support/fragments/marcAuthority/marcAuthorities';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import MarcAuthoritiesBrowseSearch from '../../../support/fragments/marcAuthority/marcAuthoritiesBrowseSearch';
+
+describe('MARC -> MARC Authority', () => {
+  const user = {};
+  const testData = {
+    marcValue: 'test',
```

Comment:
Test will fail when env is empty. Need to import marc authority before testand remove it after